### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by thomasyu888**

### CODE REVIEW

File: src/template.json [hunk 1]
```diff
@@ -38,12 +38,23 @@
...
     {
       "ansible_env_vars": [
         "ANSIBLE_SSH_ARGS='-o IdentitiesOnly=yes'"
       ],
       "playbook_file": "playbook.yaml",
       "type": "ansible"
+    },
+    {
+      "inline": [
+        "sudo systemctl enable apt-daily.timer",
```
Comment (zaro0508) [inline src/template.json]: this does not guarantee execution order with respect to the other shell provisioner defined in this file if that's important to you.. https://packer.io/docs/provisioners/shell.html#scripts

File: src/shutdown-apt-services.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+function killService() {
+    service=$1
+    sudo systemctl stop $service
+    sudo systemctl kill --kill-who=all $service
+
+    # Wait until the status of the service is either exited or killed.
+    while ! (sudo systemctl status "$service" | grep -q "Main.*code=\(exited\|killed\)")
+    do
+        sleep 10
```
Comment (zaro0508) [inline src/shutdown-apt-services.sh]: would this affect overall time required to build?  would we need to update that in .travis.yaml?

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: src/shutdown-apt-services.sh (hunk 2 above)
Comment (thomasyu888) [reply src/shutdown-apt-services.sh]: I think it's probably ok?  My timeout is 45 minutes

File: src/template.json (hunk 1 above)
Comment (thomasyu888) [reply src/template.json]: I'm not sure if this is important, this is just copied from the webpage you sent me. https://codeinthehole.com/tips/avoiding-package-lockout-in-ubuntu-1804/

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**